### ISSUE TITLE
Anatomic Site Concept ID (Entire vs Structure)

### ISSUE THREAD

**Issue by kyulee-jeon** [OHDSI/ImageWG#9]
In the MI-CDM paper (Fig3), the anatomic_site_concept_id for lung nodule use case was designated as '[4118108](https://athena.ohdsi.org/search-terms/terms/4118108) (entire thorax)'. If we obtain anatomic site information from DICOM, '[4184181](https://athena.ohdsi.org/search-terms/terms/4118108) (thoracic cavity structure)' might be more appropriate. 
p.s. I am seeking input on this matter to avoid confusion for researchers who are following MI-CDM based on our paper.

**Rationale:**
- (DICOM) DICOM generally uses anatomic concepts with "structure" rather than "entire". For instance, the BodyPartExamined Tag 'CHEST' is defined as SCT '[43799004 (thoracic cavity structure)](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html)'.
- (OMOP CDM) Both codes are included as Standard Concept IDs in the Specimen Table's [Accepted Concepts](https://ohdsi.github.io/CommonDataModel/cdm54.html#specimen), but 'Entire' lacks descendants compared to 'Structure'.

| Concept Type | OMOP Concept ID                                                     | SCT Code                                                                                                                                               | Description                                   |
| ------------ | -------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------- |
| Structure    | [4184181](https://athena.ohdsi.org/search-terms/terms/4184181) | [43799004](https://browser.ihtsdotools.org/?perspective=full&conceptId1=43799004&edition=MAIN/2024-07-01&release=&languages=en&latestRedirect=false)   | Thoracic cavity structure (body structure) |
| Entire       | [4118108](https://athena.ohdsi.org/search-terms/terms/4118108) | [302551006](https://browser.ihtsdotools.org/?perspective=full&conceptId1=302551006&edition=MAIN/2024-07-01&release=&languages=en&latestRedirect=false) | Entire thorax                                 |

**References:**
- [DICOM PS3.16 > Correspondence of Anatomic Region Codes and Body Part Examined Defined Terms](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html)
- [OMOP CDM v5.4 > Specimen Table > ETL Conventions](https://ohdsi.github.io/CommonDataModel/cdm54.html#specimen)

**Comment by jenpark10** [comment]
Agreed. We should follow the DICOM-SNOMED Value Set, which is the Thoracic cavity structure.
